lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -21,6 +21,7 @@ def self.run(params)
         automatic_release = params[:automatic_release]
         hybrid_common_version = params[:hybrid_common_version]
         versions_file_path = params[:versions_file_path]
+        include_prerelease = params[:is_prerelease]
```

Comment:
Just to check, this will be passed from CircleCI as a parameter that needs to be added manually, correct? After this is done, we probably should update notion to include these new steps for prereleases

Reply:
@tonidero Ah, so no! Nothing should need to get updated from CircleCI since this is really only meant to be run locally for one-off prerelease builds. `is_prerelease` defaults to `false` so everything should work as it does right now 💪

Reply:
We could potentially also run this from CircleCI but manually passing a parameter to the job, but that would probably be a bit more work and might not be worth it. I think we can just document how to make prerelease releases locally in notion